main.go

```diff
@@ -212,15 +212,63 @@ func populateArgs(c *cli.Context) (
 	// Canonicalize the mount point, making it absolute. This is important when
 	// daemonizing below, since the daemon will change its working directory
 	// before running this code again.
-	mountPoint, err = filepath.Abs(mountPoint)
+	mountPoint, err = getResolvedPath(mountPoint)
 	if err != nil {
 		err = fmt.Errorf("canonicalizing mount point: %w", err)
 		return
 	}
 	return
 }
 
+func getResolvedPath(filePath string) (resolvedPath string, err error) {
+	if path.IsAbs(filePath) {
+		return filePath, nil
+	}
+
+	if strings.HasPrefix(filePath, "~/") {
+		homeDir, err1 := os.UserHomeDir()
+		if err1 != nil {
+			return "", fmt.Errorf("get home dir: %w", err1)
+		}
+		return filepath.Join(homeDir, filePath[2:]), nil
+	} else {
+		return filepath.Abs(filePath)
+	}
+}
+
+func resolveRelativePath(c *cli.Context) (err error) {
+	parentProcessExecutionDir, ok := os.LookupEnv("parent_process_execution_dir")
+	if !ok { // Don't do anything, directory is not set
+		return nil
+	}
+	//var parentProcessExecutionDir string
+	logFile := c.String("log-file")
```

Comment:
We don't need this for key-file ?

Reply:
I thought, I'll take that in another CL.
But as you say, I'll take the flag-string as a parameter, which we need to resolve.